### ISSUE TITLE
Hyperlink tooltip is bounded by document canvas

### ISSUE THREAD

**Issue by kossebau** [kogmbh/WebODF#882]
When hovering over a link at the right side of the document, the tooltip gets shaped to fit in the canvas:
![Screenshot](https://cloud.githubusercontent.com/assets/4068037/6963932/c13df048-d944-11e4-9b22-b36df9c49c4d.png)

Expected:
it would float over the border of the document

**Comment by peitschie** [comment]
I would disagree that it's desirable to have the tooltip appear outside the docuemnt. After all, you probably want it to stay within the doc editor's prescribed region if possible. I would suggest it's actually that the tooltip is being pushed too far to the right.

If the tooltip expanded more to the left (where there's still plenty of room), it could appear full-size without any wrapping, and be flush against the right edge.
